runtime/proto/corfu_store_metadata.proto

```diff
@@ -19,11 +19,16 @@ message TableName {
     string table_name = 2;
 }
 
-// Descriptor to provide self describing schemas for the records.
+// Describe the protos in a file.
 message TableDescriptors {
     // File Descriptor protobufs map.
     // Contains file descriptors of key, value, metadata and their dependencies.
     map<string, google.protobuf.FileDescriptorProto> fileDescriptors = 1;
+
+    // 'Any' representing the key, value and metadata of a table
+    google.protobuf.Any key = 2;
```

Comment:
Thanks for this change. However this means that when we validate the schema we might have to force write an update into the TableRegistry even when we don't detect a direct change in the list of file descriptors, if the previous value did not have any key, value or metadata fields. This is only needed for upgraded setups.
Please look for this line... `// else no need to re-add the exact same entry into the table`

Reply:
done